app/components/profile_picture/edit_image/index.tsx

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
not sure why this component is under `profile_picture` they seem unrelated to me